examples/phase1-app/reactjs/.gitignore

```diff
@@ -0,0 +1,27 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
```

Comment:
I would probably move the content to its own new directory, the phase1-app is something that we did a long time ago, I think it should be like in "jssdkapp" or something like that cause the goal is to demonstrate that.